src/app/modules/Dashboard/Dashboard.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
-import { useQuery } from '@apollo/client';
 import { PageSection, Title } from '@patternfly/react-core';
-import { getEmployees } from '@app/graphql';
+import { GraphQL } from '../../graphql/types';
+import useGetEmployeesQuery = GraphQL.useGetEmployeesQuery;
 
 const Dashboard: React.FC = () => {
-  const { loading, error, data } = useQuery(getEmployees);
+  const results: any = useGetEmployeesQuery();
```

Comment:
```suggestion
  const  {data, error, loading} = useGetEmployeesQuery();
```
We should avoid to use `any`, use data types. The main purpose of generating types/entities to type safety and to avoid any as much as possible.